src/Mod/TechDraw/App/DrawViewSection.cpp

```diff
@@ -459,6 +457,14 @@ void DrawViewSection::sectionExec(TopoDS_Shape& baseShape)
 
     m_cuttingTool = makeCuttingTool(m_shapeSize);
 
+    if (!DU::isGuiUp()) {
```

Comment:
I guess it fixes the issue, though the real issue is the DrawViewSection in the first place assumes/depends/expects that there is a GUI.

Reply:
All the DVP derived views that run OCCT operations in separate threads need the notification that the thread has ended, and that only arrives if the Gui's event loop is available.